test/smoke/win32.test.js

```diff
@@ -15,9 +15,12 @@
 import assert from 'node:assert'
 import { test, describe } from 'node:test'
 import '../../build/globals.js'
+import { which } from '../../build/vendor.js'
```

Comment:
`import '../../build/globals.js'` goes upper, so `which` API should be available without extra import.